.changeset/tasty-windows-stick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@bifold/core': major
```

Comment:
nit: I think this should just be `patch` rather than `major`, since you are only fixing bugs not making breaking changes (major) or adding new features (minor)

Reply:
Can you guide me through this :). Its new to me